add support view

elfkuzco opened the pull request:
## Changes
- redesign support page

benoit74 reviewed (changes requested):
I feel like this page design is odd.

<img width="1129" height="487" alt="image" src="https://github.com/user-attachments/assets/ed72ee9f-dcf9-49bf-ae03-51900aad5cd9" />

While original design was too minimalist and improving it is definitely a plus, I'm puzzled by:
- the horizontal whitespaces around every links are weird
- the paragraph on donating money should be given more importance than the "Donate a worker" section; probably just invese both styles
- the fact that "Donate a worker" details is spread on two paragraphs with a big space in between is odd
- the "Donate a worker" (or its equivalent once you've moved to money) text must be vertically centered on its icon

elfkuzco commented:
removed the excess whitespaces. turns out switching to a new line for the closing anchor tag was adding the extra space. reverted back styling for `Donate a Worker` as in the first comment. Didn't like the green after much. Review and let me know if it's okay.
